pkg/scene/scan.go

```diff
@@ -201,6 +218,9 @@ func (scanner *Scanner) ScanNew(file file.SourceFile) (retScene *models.Scene, e
 
 	interactive := getInteractive(file.Path())
 
+	captions := getCaption(file.Path())
+	captioned := len(captions) > 0
```

Comment:
Since we have captions now isnt captioned obsolete?
Not just here, even in the database. Can't it be replaced with `captions != ""` for the backend/filters/ui ?

Reply:
Captioned is obsolete now. Removing it was the last thing on my to-do list once we've ironed everything else out.